Examples/Python/tests/conftest.py

```diff
@@ -153,3 +154,41 @@ def _factory(s):
         return evGen, simAlg, digiAlg
 
     return _factory
+
+
+@pytest.fixture(scope="session")
+def material_recording_session():
+    if not helpers.geant4Enabled:
+        pytest.skip("Geantino recording requested, but Geant4 is not set up")
+
+    if not helpers.dd4hepEnabled:
+        pytest.skip("DD4hep recording requested, but Geant4 is not set up")
+
+    from material_recording import runMaterialRecording
+
+    dd4hepSvc = acts.examples.dd4hep.DD4hepGeometryService(
+        xmlFileNames=["thirdparty/OpenDataDetector/xml/OpenDataDetector.xml"]
+    )
```

Comment:
Can do sure, but `conftest.py` is only relevant for the `pytest` tests, and those you can only run from the repository root anyway (because that's where `pytest.ini` is that configures it correctly)

Reply:
I'll change it anyway, it's safer.

Reply:
This should be done consistently now.